GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+# Overview
+
+This is a meritocratic, consensus-based community project. Anyone with an
+interest in the project can join the community, contribute to the project
+design and participate in the decision making process. This document describes
+how that participation takes place and how to set about earning merit within
+the project community.
+
+# Roles and responsibilities
+
+## Users
+
+Users are community members who have a need for the project. They are the most
+important members of the community and without them the project would have no
+purpose. Anyone can be a user; there are no special requirements.
+
+The project asks its users to participate in the project and community as much
+as possible. User contributions enable the project team to ensure that they are
+satisfying the needs of those users. Common user contributions include (but are
+not limited to):
+
+- evangelising about the project (e.g. a link on a website and word-of-mouth
+awareness raising)
+- informing developers of strengths and weaknesses from a new user perspective
+- providing moral support (a ‘thank you’ goes a long way)
+
+Users who continue to engage with the project and its community will often
+become more and more involved. Such users may find themselves becoming
+contributors, as described in the next section.
+
+## Contributors
+
+Contributors are community members who contribute in concrete ways to the
+project. Anyone can become a contributor, and contributions can take many
+forms. There is no expectation of commitment to the project, no specific skill
+requirements and no selection process.
+
+In addition to their actions as users, contributors may also find themselves
+doing one or more of the following:
+
+- supporting new users (existing users are often the best people to support
+new users)
+- reporting bugs
+- identifying requirements
+- providing graphics and web design
+- programming
+- assisting with project infrastructure
+- writing documentation
+- fixing bugs
+- adding features
+
+Contributors engage with the project through the issue tracker and mailing
+list, or by writing or editing documentation. They submit changes to the
+project itself via patches, which will be considered for inclusion in the
+project by existing maintainers (see next section). The developer mailing list
+is the most appropriate place to ask for help when making that first
+contribution.
+
+As contributors gain experience and familiarity with the project, their profile
```

Comment:
This paragraph leaves the question about who can nominate someone. Also `above-average` seems to be strange. If we have three maintainers and one contributor, the contributor needs to contribute more that the two of the maintainers to achieve an above-average contribution level. If the maintainers do not count, then a single contributor cannot contribute more than average. Therefore I would just remove this.

Reply:
I think that the intent here is to provide a path and a vague definition for a contributor to be promoted. I think it looks reasonable enough. Contribution is not limited to commits.

Reply:
Hm. AFAIK, there is no a "way" to become a maintainer in any project. Normally, someone from the core members of the project (maintainers in Nmstate) propose a contributor as a new maintainer and if everyone (or a majority) agree, then they officially invite him/her to join the project as a maintainer.

Also, IMHO it is not only about `above-average contributions` because it is also very important the behaviour of that contributor with the community.